permissions/src/iosMain/kotlin/BluetoothPermissionManager.kt

```diff
@@ -0,0 +1,61 @@
+package com.splendo.mpp.permissions
+
+import platform.CoreBluetooth.CBCentralManager
+import platform.CoreBluetooth.CBPeripheralManager
+import platform.Foundation.NSURL
+import platform.UIKit.UIApplication
+
+class BluetoothPermissionManager(
+    private val cbCentralManager: CBCentralManager,
+    private val cbPeripheralManager: CBPeripheralManager
+) : PermissionManager() {
+
+    override suspend fun openSettings() {
+        UIApplication.sharedApplication.openURL(NSURL(string = "App-Prefs:root=General"))
+    }
+
+    override suspend fun checkSupport(): Support {
+        return when (cbCentralManager.state.toLong()) {
+            STATE_UNKNOWN -> Support.NOT_SUPPORTED
+            STATE_RESETTING -> Support.RESETTING
+            STATE_UNSUPPORTED -> Support.NOT_SUPPORTED
+            STATE_UNAUTHORIZED -> Support.UNAUTHORIZED
+            STATE_POWERED_OFF -> Support.POWER_OFF
+            STATE_POWERED_ON -> Support.POWER_ON
+            else -> Support.NOT_SUPPORTED
+        }
+    }
+
+    override suspend fun checkPermit(): Permit {
+        //TODO: Add support for iOS 13. `authorizationStatus` is being deprecated
```

Comment:
Can you make an issue for this and add the link or number here?

Reply:
https://github.com/splendo/kaluga/issues/21